packages/v4/src/content/design-guidelines/components/switch/switch.md

```diff
@@ -6,24 +6,26 @@ A **switch** toggles the state of a setting (between on and off). Switches and [
 
 ## Usage
 
-It is assumed that when a user toggles a switch, the change will save automatically without the need for additional action (like clicking a “Save” button).
+It is assumed that when a user toggles a switch, the change will save automatically without the need for additional action (like clicking a "Save" button).
 
 <img src="./img/switch.png" alt="Examples of switch buttons" width="244"/>
 
 **General guidelines**
 
 *  Add a label in front of your switch to specify what is being switched on or off (for example, wi-fi or bluetooth).
-* Don’t use a switch if the options you’re presenting to the user are anything other than “on” or “off.” Instead, use radio buttons. 
-* Depending on your use case, you may want to add labels to your switches for added clarity, like “on” and “off” to indicate to the user what setting they’ve switched to.
+* Don’t use a switch if the options you’re presenting to the user are anything other than “on” or “off.” Instead, use [radio buttons](/components/radio/design-guidelines). 
+* Depending on your use case, add labels to clarify between toggled settings, such as *On* and *Off*.
 
 
 ### When to use switches versus checkboxes
 
-The main difference between a switch and a checkbox is that a switch changes an option and saves it simultaneously, while checkboxes require a separate action such as pressing a “Submit” or “Save” button to save the selection. 
+A switch changes an option and saves it simultaneously, while checkboxes require a separate action to save the selection, such as pressing a “Submit” or “Save” button.
 
-**Here are some guidelines for when to use a switch versus a checkbox:**
+Use switches with labels by default unless space is limited, then use the checkmark version instead.
```

Comment:
Can you move this, and make it a bullet under "General guidelines" instead? It doesn't make sense here because it's not talking about "checkboxes" it's still just talking about the switcher component itself!

Reply:
And maybe also to clarify, just say "Use switches with labels by default unless space is limited, then use a switch without a label" and link "Switch without a label" to this: https://patternfly-org-pr-2287-v4.surge.sh/v4/components/switch/react#without-label - thank you!!!